test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -12,6 +12,9 @@
     <!-- Globbed link tag with existing file -->
     <link asp-href-include="**/site.css" rel="stylesheet" title="&lt;the title>" />
 
+    <!-- Globbed link tag with comma separated pattern -->
+    <link asp-href-include="~/*.css, ~/sub/*.css" rel="stylesheet" title="&lt;the title" />
```

Comment:
@NTaylorMullen not sure about _functional_ variations since that sounds a lot like the cross-product of inputs and options that should be covered in unit tests. These tag helper functional tests area already stray a bit too much from mainline scenarios. But including more whitespace in existing functional tests (not creating new cases) sounds reasonable.